contracts/solidity/control/ControlStructures.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+contract TestContract {
+
+    constructor() {}
+
+    function myFunc(uint x) public pure returns (string memory) {
+        require(x != 0, "require failed");
+        return "my func was called";
+    }
+}
+
+
+contract ControlStructures {
+    TestContract public test;
```

Comment:
Why public? If we don't use it out of the `ControlStructures` contract in the tests, let's make it private/internal.

Reply:
and rename it to `testContract` :shrug: